components/search_bar/search_bar.jsx

```diff
@@ -84,7 +84,11 @@ export default class SearchBar extends React.Component {
     }
 
     handleUserBlur = () => {
-        this.setState({focused: false});
+        // add time out so that the pinned and member buttons are clickable
+        // when focus is released from the search box.
+        setTimeout(() => {
+            this.setState({focused: false});
+        }, 100);
```

Comment:
Not sure I understand what the timeout is being used for. Is it because we're in a render cycle and for some reason this change loses in a race or doesn't get picked up?

Reply:
Yes, I think what's happening currently is that the blur event always occurred first before the click event. 
This will somehow reverse the order. (Actually, setting that to 0 will still not do the trick.)

Reply:
Yeah, if you change the transition length from `0.3s` to about `1s` on `.search-bar__container .search__form`, the click event manages to fire on the position of the clicked element before the search bar resizes and moves it out of the way.

Another, still somewhat hacky way might be to handle mouse down events on the buttons instead of click events, since the event order seems to be `mousedown > blur > mouseup > click`. Otherwise, I don't see any way around this besides the timeout proposed here.